### PR TITLE
Qrexec internals further work

### PR DESCRIPTION
**Pull request by pierwill**
Continue working on qrexec internals docs (following #870).

### CODE REVIEW

File: developer/services/qrexec-internals.md [hunk 1]
```diff
@@ -32,26 +40,41 @@ Because of [vchan limitation](https://github.com/qubesos/qubes-issues/issues/951
...
   * `cmdline`: Command line to pass to `qrexec-daemon` as the execution/service request. Service request format is described below in the service example.
 
-**Note:** None of the above tools are designed to be used by users directly.
-
 ## VM tools implementation
 
-### `qrexec-agent`: One instance runs in each active domain. Responsible for:
+### `qrexec-agent`
+
+<!-- PATH? -->
```
Comment (marmarek) [inline developer/services/qrexec-internals.md]: `/usr/lib/qubes/qrexec-agent`

File: developer/services/qrexec-internals.md [hunk 2]
```diff
@@ -32,26 +40,41 @@ Because of [vchan limitation](https://github.com/qubesos/qubes-issues/issues/951
...
+One instance runs in each active domain. 
+Responsible for:
   * Handling service requests from `qrexec-client-vm` and passing them to connected `qrexec-daemon` in dom0.
   * Executing associated `qrexec-daemon` execution/service requests.
 * Command line parameters: none.
 
-### `qrexec-client-vm`: Runs in an active domain. Used to pass service requests to `qrexec-agent`.
+### `qrexec-client-vm`
+
+<!-- PATH? -->
```
Comment (marmarek) [inline developer/services/qrexec-internals.md]: `/usr/bin/qrexec-client-vm`

**Comment by andrewdavidwong** [comment]
Couple of typos: https://travis-ci.com/QubesOS/qubes-doc/builds/142061258#L4596-L4603

**Comment by pierwill** [comment]
@andrewdavidwong  Thanks, Andrew! Pushed those changes. There's more work to do on the internals docs still, but I'm thinking it makes sense to merge this PR.

**Pull request merged**